battleground_viewer/src/construct_render/render.rs

```diff
@@ -0,0 +1,408 @@
+use three_d::*;
+
+use battleground_construct::display::primitives::Primitive;
+
+// Brend: This render pass enumeration omits non-geometry passes (such as the bloom filter application, or the gui render).
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub enum RenderPass {
+    ShadowMaps,
+    BaseScene,
+    NonEmmisivesDepth,
```

Comment:
Is `NonEmmisivesDepth` related to `Emissives` or is it the generic material pass? If it is a generic render pass, the name should represent that and it should not be named as `NotTheFancyEffect` if it has nothing to do with `FancyEffect`.

While we're at it... can we just document each pass with `///` and give them all a sentence to explain what should go into that pass?

Reply:
`NonEmmissivesDepth` is indeed related to `Emissives`.